packages/pigment-css-nextjs-plugin/package.json

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
Oh, pigment is still in master cc @brijeshb42

Reply:
See https://github.com/mui/material-ui/pull/42897#discussion_r1673888676

Reply:
https://github.com/mui/material-ui/pull/42897#discussion_r1673888676